plugin/src/main/java/net/codingarea/challenges/plugin/challenges/implementation/challenge/RandomPotionEffectChallenge.java

```diff
@@ -77,6 +78,13 @@ public static PotionEffectType getNewRandomEffect(@Nonnull LivingEntity entity)
 		return possibleEffects.get(IRandom.threadLocal().nextInt(possibleEffects.size()));
 	}
 
+	@Nullable
+	public static PotionEffectType getRandomEffect() {
```

Comment:
Please don't create static methods into challenge classes also when you don't even use this method.